systemdev/calendars.rst

```diff
@@ -0,0 +1,54 @@
+Working on the Calendar feature
+===============================
+
+The Calendar feature is provided by several components:
+
+* The `Calendar App <https://gitlab.com/ubports/apps/calendar-app>`_, providing the user interface
+
+* `Evolution Data Server <https://github.com/ubports/evolution-data-server-packaging>`_ (often shortened as "EDS"), which is the backend where your calendars are stored
+
+* `sync-monitor <https://github.com/ubports/sync-monitor>`_, the service responsible for managing the synchronisation with remote calendars
+
+* `SyncEvolution <https://github.com/ubports/syncevolution-packaging>`_, the service responsible for performing the synchronisation to a WebDAV/CalDAV remote calendar
```

Comment:
I wasn't aware that you could have blank lines between bullet points and keep them in a single list. Even so, this is the only file that does this. Would you mind removing the blank line to keep the style?

Reply:
done